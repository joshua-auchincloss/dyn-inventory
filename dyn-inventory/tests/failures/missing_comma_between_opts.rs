// missing comma between two ExtraParams
//
//
//
//
//
//
//
//
//
use dyn_inventory::dyn_inventory;

dyn_inventory!(
    CommaIssueRegistry<T: PluginApi> {
        size: i16,
        t: T,
    };
    init_name = SomeInit
    unknown = RegisterTwo,
);

fn main() {}
