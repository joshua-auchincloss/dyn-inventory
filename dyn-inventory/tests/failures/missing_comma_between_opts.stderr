error: expected `,`
  --> tests/failures/missing_comma_between_opts.rs:19:5
   |
19 |     unknown = RegisterTwo,
   |     ^^^^^^^
